test/mocha/30-event-api.js

```diff
@@ -12,28 +12,27 @@ const mockData = require('./mock.data');
 const uuid = require('uuid/v4');
 
 const exampleLedgerId = 'did:v1:' + uuid.v4();
-const configEventTemplate = mockData.events.config;
-const configBlockTemplate = mockData.configBlocks.alpha;
+const configEventTemplate = _.cloneDeep(mockData.events.config);
+configEventTemplate.ledger = exampleLedgerId;
 
+const configBlockTemplate = _.cloneDeep(mockData.configBlocks.alpha);
```

Comment:
I've seen `_.cloneDeep` creeping into code.  Does this do something different/better that `bedrock.util.clone`?

Reply:
Copied from elsewhere in the test suite in this case. We should probably just start phasing out `bedrock.util.clone` so we don't have to maintain or try and improve the clone method.

Reply:
bedrock already has a `lodash` dep, should we just alias `bedrock.util.clone` to `_.cloneDeep`?

Reply:
That's step one, yeah. We should make sure there's nothing "better" or "special" about what `bedrock.util.clone` is doing first though (I doubt there is anything).